### PR TITLE
FEAT: Openr Environment arg

### PR DESCRIPTION
**Pull request by caiopgarcia**
## Description
Change to introduce OPENR_ENVIRONMENT as argument for Flyway

## Linked PRs
https://github.com/weareopenr/api/pull/1828

### CODE REVIEW

**Review by the-alex-openr — APPROVED** [review]

File: action.yaml [hunk 1]
```diff
@@ -40,6 +40,9 @@ inputs:
     description: Which action flyway will perform (ie. migrate or repair)
     required: true
     default: migrate
+  environment:
+    description: The Openr environment (dev, tech, prod)
```
Comment (Smittey) [inline action.yaml]: Can we use stag or staging instead of tech? Staging is consistent throughout all other repos

**Comment by Smittey** [comment]
Could you also update this line in the readme? `https://github.com/weareopenr/flyway-action/blob/master/README.md?plain=1#L42` - it leads to a 404. Also I think the readme could do with including the new custom attributes

File: action.yaml (hunk 1 above)
Comment (Smittey) [reply action.yaml]: Maybe default to dev?

Comment (Smittey) [reply action.yaml]: Also though, see corresponding comment: https://github.com/weareopenr/api/pull/1828/files#r1617015683

Comment (caiopgarcia) [reply action.yaml]: Yeah, this is will actually be used to generate endpoint types in the db according to our deployment naming convention.
As an example: `https://api.marketplace.tech/fourth/recipes`

Comment (caiopgarcia) [reply action.yaml]: @Smittey , this is how we are resolving it: https://github.com/weareopenr/api/pull/1828/files#:~:text=CASE%20%27%24%7Benvironment%7D%27

I just made sure this isn't a required arg too

**Review by peterholdsworth — APPROVED** [review]

**Pull request merged**